run.py

```diff
@@ -18,10 +19,20 @@
     ))
 
     argparser = argparse.ArgumentParser()
+    argparser.add_argument("filename", type=str, help="Name of the output file")
     argparser.add_argument("-j", "--json", action="store_true",
                            help="Choose if the statistics will be saved to a json file")
+    argparser.add_argument("-g", "--global_stats", action="store_true")
+    argparser.add_argument("-c", "--country_stats", action="store_true")
 
     args = argparser.parse_args()
+    current_date = datetime.now().strftime("%m_%d_%y")
+    filename = "statistics_" + current_date + '.json'
 
+    if len(args.filename.strip()) > 0:
+        filename = args.filename + '.json'
     if args.json:
-        ex.export_to_json(ct.all_statistic, ct.country_statistic)
+        if args.global_stats:
+            ex.export_to_json('global_' + filename, ct.all_statistic)

```

Comment:
The reason that there is added a prefix ("global_" or "country_") is because JSON standard does not allow two dictionaries in one file. Therefore, if the user wants to save both dictionaries, two files are needed.

Reply:
How about merging two data(all_statistic, country_statistic) into one json file after receiving the file name from the user? :)
```
{
  "all": {
    ... all_statistic data ...
  },
  "contries": {
    ... country_statistic data ...
  }
}
```

Reply:
I gave it a try: 1378363